torch_geometric/utils/get_mesh_laplacian.py

```diff
@@ -0,0 +1,73 @@
+import torch
+from torch_scatter import scatter_add
+
+from torch_geometric.utils import add_self_loops
+
+
+def get_mesh_laplacian(pos, face):
+    """ Computes the mesh Laplacian of the mesh given by
+    :obj:`pos` and :obj:`face`. It is computed as
+    :math:` \mathbf{L}_{ij} = \begin{cases} 

```

Comment:
Use math environment rather than in-place formula
```suggestion
    .. math::
        \mathbf{L}_{ij} = \begin{cases} 
```

Reply:
Great, thank you!